### PR TITLE
Feat/pit send dropdown

### PR DESCRIPTION
**Pull request by plondon**
https://blockchain.atlassian.net/browse/WEB4-2512

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagaRegister.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import * as AT from './actionTypes'
 import { actionTypes } from 'redux-form'
 import sagas from './sagas'
 
-export default ({ coreSagas, networks }) => {
-  const sendBtcSagas = sagas({ coreSagas, networks })
+export default ({ api, coreSagas, networks }) => {
+  const sendBtcSagas = sagas({ api, coreSagas, networks })
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagaRegister.js]: `api` doesnt appear to be used

File: packages/blockchain-wallet-v4-frontend/src/data/components/sagaRegister.js [hunk 2]
```diff
@@ -53,8 +54,9 @@ export default ({ api, coreSagas, networks }) =>
     yield fork(requestBch({ networks }))
     yield fork(requestEth({ networks }))
     yield fork(requestXlm())
+    yield fork(send({ api }))
     yield fork(sendBch({ coreSagas, networks }))
-    yield fork(sendBtc({ coreSagas, networks }))
+    yield fork(sendBtc({ api, coreSagas, networks }))
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/sagaRegister.js]: `api` doesnt appear to be used

File: packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js [hunk 3]
```diff
@@ -71,6 +72,13 @@ export const getData = (state, ownProps) => {
   const excluded = filter(x => !exclude.includes(x.label))
   const toDropdown = map(x => ({ label: buildDisplay(x), value: x }))
   const toGroup = curry((label, options) => [{ label, options }])
+  const toPit = x => [{ label: `BCH PIT Address (${x})`, value: x }]
+
+  const pitAddress = selectors.components.send.getPaymentsAccountPit(
+    'BCH',
+    state
+  )
+  const hasPitAddress = Remote.Success.is(pitAddress)
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js]: 👍

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js]: Assuming this safely allows the send dropdown to still render even if API call fails

**Comment by schnogz** [comment]
Do we want to add a higher check to see if they even have PIT link before calls?

File: packages/blockchain-info-components/src/Buttons/Button.js [hunk 4]
```diff
@@ -180,6 +180,7 @@ Button.propTypes = {
     'gray-3',
     'light',
     'primary',
+    'purple',
```
Comment (CourseZa) [inline packages/blockchain-info-components/src/Buttons/Button.js]: 1MoSVa1zdMBXBuzwo5H5sA7yZZxUmxnT9x

Comment (CourseZa) [reply packages/blockchain-info-components/src/Buttons/Button.js]: 15seQHK2njucEAjUJvyfwFmH9tMWeUNEBK

**Pull request merged**